### DISCUSSION TITLE
[Bug]: UI 0.14 Event review scrub progress bar auto plays constantly

### DISCUSSION THREAD

**Discussion by ikout** [blakeblackshear/frigate#12793] [Report a Bug] [answered]
### Describe the problem you are having

When you are at the review page and you hover an event it automatically plays a low res clip and if you go at the bottom of the image you can scrub the clip manually instead of the auto-play. 
Issue is that once you start the scrub, by moving the mouse at the same line or even clicking at the scrub line and moving it, if you then stop for a second or two the auto-play kicks in again and continues to loop the preview.

Once you click or are in the scrub line the auto-play should stop/pause until the mouse moves away from the preview.

### Steps to reproduce

1. Go to the events page.
2. Move the mouse to an event, it will auto-play a preview.
3. Move the mouse at the lower part of the preview to activate the scrub line. (Or if you click on the scrub line)
4. Use the scrub line and move to a place in the preview and stop the mouse for  1-2 seconds.
5. The auto-play will start again and you will lose the position that you are.
6. Also if you click on the scrub line and do the same it will "jitter" back and forth to the next frame one the auto-play and the place that you have stopped the mouse

### Version

0.14.0-da913d8

### Frigate config file

```yaml
No special config needed
```

### Relevant log output

```shell
No logs needed
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Mixed

### Camera make and model

Tp-Link

**Comment by NickM-27** [comment, marked as answer]
Thanks, fixed in https://github.com/blakeblackshear/frigate/pull/12794
